packages/datetime/src/dateFormat.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { isDateValid, isDayInRange } from "./common/dateUtils";
+import { IDatePickerBaseProps } from "./datePickerCore";
+
+export interface IDateFormatProps {
+    /**
+     * Date format string, passed to `formatDate` and `parseDate`.
+     */
+    format?: string;
```

Comment:
@adidahiya what's your take on this prop? worth keeping?

Reply:
prefer to keep it simple and add it later if it's desired